Source/WebCore/platform/graphics/ca/TileGrid.cpp

```diff
@@ -713,6 +713,10 @@ void TileGrid::drawTileMapContents(CGContextRef context, CGRect layerBounds) con
 
 void TileGrid::platformCALayerPaintContents(PlatformCALayer* platformCALayer, GraphicsContext& context, const FloatRect&, OptionSet<GraphicsLayerPaintBehavior> layerPaintBehavior)
 {
+#if USE(CA)
+    if (containerLayer().owner() && containerLayer().owner()->platformCALayerRenderingSuppressed())
```

Comment:
It feels wrong to implement rendering suppression by short circuiting painting; this makes me nervous. We should have just not make the layers as needing display.

What's the expected behavior when you resize the window while rendering is suppressed? Do we need to be able to paint the old state on layer resize?

Reply:
Window resizing aborts the view transition and should remove suppression so that we can repaint.

There's no expectation to be able to paint the old state again, since that would be wildly difficult.